awsx/schema-types.ts

```diff
@@ -346,6 +346,7 @@ export interface NetworkLoadBalancerArgs {
     readonly name?: pulumi.Input<string>;
     readonly namePrefix?: pulumi.Input<string>;
     readonly preserveHostHeader?: pulumi.Input<boolean>;
+    readonly securityGroups?: pulumi.Input<pulumi.Input<string>[]>;
```

Comment:
Interesting that this is added but consuming code didn't have to have any changes. Feels a bit magical.

Reply:
I did check the different language SDKs to be sure, but all of them seem to be able to correctly handle adding optional types without breaking changes.